src/CommentsCount.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import FacebookProvider, {Like, Share} from 'react-facebook';

```

Comment:
@MrNice42 Same here... ( To fix broken build, use relative imports instead importing from 'react-facebook' ).

```js
import FacebookProvider from './FacebookProvider';
import Like from './Like';
import Share from './Share';
```